Docs/reference/content/reference/driver/crud/client_side_encryption.md

```diff
@@ -0,0 +1,206 @@
++++
+date = "2019-09-30T20:38:-04:00"
+title = "Client Side Encryption"
+[menu.main]
+  parent = "Reference Reading and Writing"
+  identifier = "Client-Side Field Level Encryption"
+  weight = 40
+  pre = "<i class='fa fa-lock'></i>"
++++
+
+# Client-Side Field Level Encryption
+
+New in MongoDB 4.2, client-side field level encryption allows administrators and
+developers to encrypt specific data fields in addition to other MongoDB
+encryption features.
+
+With client-side field level encryption, developers can encrypt fields
+client-side without any server-side configuration or directives. Client-side
+field level encryption supports workloads where applications must guarantee that
+unauthorized parties, including server administrators, cannot read the encrypted
+data.
+
+{{% note class="important" %}} 
+Support for client-side field level encryption is in beta, and the beta only
+supports Windows. Backwards-breaking changes may be made before the
+final release (which will support MacOS and Linux).
+{{% /note %}}
+
+## Installation
+
+The recommended way to get started using client-side field level encryption in
+your project is with a package manager such as NuGet. Client-side field level
+encryption requires one additional package, `MongoDB.Libmongocrypt`, in addition
+to the standard driver packages.
```

Comment:
Perhaps a rephrase?

The recommended way to get started using client-side field level encryption in your project is with a package manager such as NuGet.  Compared to previous releases, client-side field level encryption requires the driver to depend on one additional package, MongoDB.Libmongocrypt`. This dependency will be taken care of automatically when using a package manager.

Reply:
My question would be do we need to say anything?

There is nothing different about using the driver (at least from a Nuget perspective) just because you are using FLE.

Reply:
I'm happy to strike this section. Removing it!

Reply:
(I've also updated the heading levels for the rest of the document after striking this section, setting them all to level two. (##)